File: generator/src/main/java/ch/bailu/gtk/tag/MemberTag.java
package ch.bailu.gtk.tag;

public class MemberTag extends ParameterTag {
    public MemberTag(Tag parent) {
        super(parent);
    }

    public String getType() {
        if (super.getType() == null)
            return "int";

        return super.getType();
    }

    @Override
    public String getName() {
        return super.getName().toUpperCase();
    }
}

File: generator/src/main/kotlin/ch/bailu/gtk/tag/MemberTag.kt
package ch.bailu.gtk.tag

class MemberTag(parent: TagWithParent) : ParameterTag(parent) {

    override fun getType(): String {
        return if ("" == super.getType()) "int" else super.getType()
    }

    override fun getName(): String {
        return super.getName().uppercase()
    }
}
